target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -165,6 +165,11 @@ wn3000rpv3)
 	ucidef_set_led_default "power_r" "POWER (red)" "$board:red:power" "0"
 	set_wifi_led "$board:green:router"
 	;;
+ex3700)
+	ucidef_set_led_default "power_a" "POWER (amber)" "$board:amber:power" "0"
```

Comment:
The blinking amber led on bootup is used by the stock firmware on this device, so this behaviour was retained for LEDE.